doc/devel/documenting_mpl.rst

```diff
@@ -476,8 +467,52 @@ Use abbreviated links ```.Normalize``` in the text.
 
 .. code-block:: rst
 
-  norm : `~matplotlib.colors.Normalize`, optional
-     A `.Normalize` instance is used to scale luminance data to 0, 1.
+   norm : `~matplotlib.colors.Normalize`, optional
+        A `.Normalize` instance is used to scale luminance data to 0, 1.
+
+Default values
+~~~~~~~~~~~~~~
+As opposed to the numpydoc guide, parameters need not be marked as
+*optional* if they have a simple default. This removes unnecessary clutter.
+The optional aspect is already clear from the presence of a default value.
```

Comment:
I think "This removes unnecessary clutter. The optional aspect is already clear from the presence of a default value." gets into unnecessary value judgement territory and the guidelines are clear enough without it.

Reply:
Do you propose to leave that out completely? My motivation for that was to explain why we deviate from numpydoc.

It‘s been in before and just moved down a bit. But I‘m happy to improve on the wording.

Reply:
Yeah, I think it can be cut. The examples illustrate the reasoning.

Reply:
Sounds fair. Thanks for the feedback!